src/Analyzers/CSharp/CodeFixes/UsePrimaryConstructor/CSharpUsePrimaryConstructorCodeFixProvider.cs

```diff
@@ -268,12 +268,12 @@ ParameterListSyntax UpdateReferencesToNestedMembers(ParameterListSyntax paramete
                     getElements(list),
                     (p, _) =>
                     {
-                        var parameterLeadingWhitespace = GetLeadingWhitespace(p);
-                        if (parameterLeadingWhitespace.EndsWith(indentation))
+                        var elementLeadingWhitespace = GetLeadingWhitespace(p);
+                        if (elementLeadingWhitespace.EndsWith(indentation))
```

Comment:
renamed to be more accurate.  this helper was used for more htan just parameters.